app/src/main/java/com/github/se/eduverse/ui/folder/DisplayFolder.kt

```diff
@@ -83,7 +83,7 @@ fun FolderScreen(
   Scaffold(
       modifier = Modifier.testTag("scaffold"),
       topBar = {
-        TopNavigationBar(activeFolder!!.name, navigationActions) {
+        TopNavigationBar(navigationActions) {
```

Comment:
Here we should still be able to see the name of the folder (see comment on TopNavigationBar)